spec/views/posts/show_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'Post Details Section', type: :feature do
+  before do
+    @user = User.create(name: 'Tom')
+    @post = Post.create(author: @user, title: 'Hello',
+                        text: 'This is my first post')
+    Comment.create(user: @user, post: @post, text: 'Great lesson for everyone')
+    visit user_post_path(@post.author, @post)
+  end
+
+  it 'displays the post details and comments' do
+    within('.post-details') do
+      expect(page).to have_selector('.user-count span', text: 'Comments: 1')
+      expect(page).to have_selector('.user-count span', text: 'Likes: 0')
+      expect(page).to have_selector('h2', text: @post.title)
+      expect(page).to have_content(@user.name)
+    end
+
```

Comment:
- [ ] You are missing a few test cases for the posts show page. Kindly add these as demanded by project requirements.
  - A test case to check if the post body is displayed
  - Check if each comment is display
  - Check if the username of a commenter for each comment is displayed

Reply:
All tests added. Thanks